feat: store chrome/libwebrtc logs in chrome_debug.log file

mariogasparoni opened the pull request:
The file is located at the debug folder (/tmp/debug-folder-XY/chrome_debug.log)

mariogasparoni committed:
feat: store chrome/libwebrtc logs in chrome_debug.log file

The file is located at the debug folder (/tmp/debug-folder-XY/chrome_debug.log)

TiagoJacobs commented on Dockerfile:
```diff
@@ -15,6 +15,7 @@ RUN apt-get update \
     && rm -rf /var/lib/apt/lists/*
 
 ENV PUPPETEER_SKIP_CHROMIUM_DOWNLOAD true
+ENV CHROME_LOG_FILE /debug/chrome_debug.log
```
can you try to pass this variable directly from `index.js`?

before calling microphoneBrowser:
```
process.env.CHROME_LOG_FILE = `/debug/${test_id}-microphone-chrome-debug.log`;
```

before calling listenOnlyBrowser:
```
process.env.CHROME_LOG_FILE = `/debug/${test_id}-listenonly-chrome-debug.log`;
```

mariogasparoni replied:
Sure !

mariogasparoni replied:
I had to do a bit different, because process.env.CHROME_LOG_FILE was being called with the same value for both tests. So i used 'env' param.

TiagoJacobs replied:
cool!

mariogasparoni committed:
fix: separate microphone and listenonly chrome_debug.log into different files

chrome_debug.log file for microphone is located at
/tmp/debug-folder-x/test_id-microphone-chrome-debug.log

chrome_debug.log file for listenonly is located at
/tmp/debug-folder-x/test_id-listenonly-chrome-debug.log
